tests/robot-tests/tests/visual_testing/visually_check_tables_and_charts.template.robot

```diff
@@ -0,0 +1,23 @@
+*** Settings ***
+Resource            tables_and_charts.robot
+Library             tables_and_charts.py
+
+Force Tags          VisualTesting    GeneralPublic    Local    Dev    Test    Preprod
+
+Suite Setup         do suite setup
```

Comment:
Just wondering what `do suite setup` is here?

Reply:
```
    generate releases    {{datablocks_csv_filename}}
    user opens the browser
```

So it just reads the Releases in from the CSV before starting everything